DesignHashSet

```diff
@@ -0,0 +1,30 @@
+class MyHashSet {
```

Comment:
Follow this template 
// Time Complexity : 
// Space Complexity :
// Did this code successfully run on Leetcode : 
// Any problem you faced while coding this :
// Your code here along with comments explaining your approach